vocab_list/models.py

```diff
@@ -283,7 +283,7 @@ def data(self):
             headword=self.headword,
             gloss=self.gloss,
             familiarity=self.familiarity,
-            node=self.node.pk if self.node is not None else None,
+            node=self.node_id if self.node_id is not None else None,
```

Comment:
Woah, did this take care of it? That's awesome.

Reply:
yea it did! Still unsure about why there was calls to the lattice node table if the node_id was already present on the object.

Reply:
Its not totally clear to me either but maybe by calling `self.node.pk` you are instantiating a node object and then getting the pk, where calling `self.node_id` is using the id that is used for the foreign key relationship. So in the latter, there is no query required. I've convinced myself actually.